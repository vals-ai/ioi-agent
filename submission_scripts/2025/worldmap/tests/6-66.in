version https://git-lfs.github.com/spec/v1
oid sha256:6b774516fbd7085ebc48787f9f4e4cd96f6ec8308d330ff3425f513c198929ae
size 7606
