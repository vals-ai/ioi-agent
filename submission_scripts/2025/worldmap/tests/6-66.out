version https://git-lfs.github.com/spec/v1
oid sha256:9380cbee17c1a2cd9aab31503a97a76e82c2e0bd171abbec832318290ea2b2de
size 190190
